### ISSUE TITLE
"Output Names" supporting custom paths

### ISSUE THREAD

**Issue by t3dotgg** [mifi/lossless-cut#1532]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

tl;dr - would be awesome to use variables to define custom folders as part of output name.

I would love to have all the clips with a given segment name to be grouped in a single folder. Might be down to work on this on stream, even more down to put a bounty on it if y'all are cool with that

![image](https://user-images.githubusercontent.com/6751787/229255601-a226366c-922b-4a75-8c20-25bc7ffd4ec1.png)

**Comment by mifi** [comment]
I'll have a stab at it

**Comment by t3dotgg** [comment]
You are a legend @mifi

**Comment by mauticom** [comment]
Where can I add/modify these variables? I just upgraded to 3.54 and don't see this preview setup on the export preview page nor elsewhere.

**Comment by mifi** [comment]
Disable simple mode, then enable export confirmation dialog.

**Comment by mauticom** [comment]
Thanks. I was on advanced but after your hint I clicked on the output filename and then the preview and vars appeared as shown in the screenshot. Hidden feature 😉.

**Comment by mauticom** [comment]
Update: on macOS you can only enter \ instead of / and it doesn't create folders it just adds the directory part \ to the Filename in front.

**Comment by mifi** [comment]
yes I think it could use some better documentation. now it's kind of like with Apple products, you just have to know that shaking the phone is undo.
